core-galleon-pack/src/main/resources/feature_groups/audit.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<feature-group-spec name="audit" xmlns="urn:jboss:galleon:feature-group:1.0">
```

Comment:
This one perhaps should be name management-audit as there's other audit functionality in the server.

@darranl WDYT?

Reply:
Renaming to management-audit.